features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/Utils.js

```diff
@@ -183,6 +212,7 @@ class Utils {
 Utils.CONST = {
     LOCAL_STORAGE_ENVIRONMENT: 'environment_publisher',
     DCR_APP_INFO: '/login/login',
+    AUTO_LOGIN_ENABLED_INFO: 'auto_login',
```

Comment:
Didn't we remove auto_login variable name as discussed earlier? This should be something multi-environment overview related.

Reply:
I used this to store the config in localstorage
- Set auto login (multi-environment overview) configs in local storage
![image](https://user-images.githubusercontent.com/23183042/36149255-17595cf8-10e5-11e8-82fc-3d09b2370a87.png)